tests/unit/test_docker_registries_watcher.py

```diff
@@ -89,3 +89,24 @@ async def test_docker_registries_watcher(
         "ubuntu": "image signature changed",
     }
     _assert_docker_client_calls(mock_docker_client, registry_config, valid_docker_stack)
+
+    # Handle the failure of fetching an image
```

Comment:
very nice test!
Just a small suggestion:  i would create an independent test for this e.g. ``test_handle_failure_of_fetching_an_image``. It much better to have a lot of small unit tests with a very clear target than having one big-fat with a lot of checks and different contexts. If there are common parts, you can create helper functions or use the fixture.
Also mockups are typically part of the test setup, which is included in the fixture although when the test is small it is also acceptable to do it in the test-function body